data/places.ts

```diff
@@ -0,0 +1,37 @@
+/* This is just a dummy file to test that the db connection works 
+The content of this file should be changed later when we start writing database functions
+*/
```

Comment:
When I see the `data` folder, I don't know if that's getting data or actual data files. I would recommend making a new folder in `src` called `db` and then moving this file there.

Reply:
I like that, will do 👍🏻